Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'ya2yaml'
 gem 'i18n'
 gem 'dynamic_form'
 gem 'exception_notification'
-gem 'activerecord-import'
+gem 'activerecord-import', '>= 0.2.0'
```

Comment:
You can probably revert this change. We're using activerecord-import in a pretty limited way.